setup.py

```diff
@@ -22,10 +22,11 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 from setuptools import setup
+from s3po import __version__
```

Comment:
To be clear you can still import the version constant from the version file into the main package so that it is available there as well.